e2e-tests/insurance/cypress/e2e/journeys/account/account-creation-and-sign-in-with-eligibility-answers.spec.js

```diff
@@ -58,6 +61,14 @@ context('Insurance - Account - When answering eligibility answers, creating an a
     });
 
     it(`should redirect to ${dashboardUrl} and have one application in the table`, () => {
+      cy.getReferenceNumber().then((refNumber) => {
+        const expectedUrl = `${baseUrl}${ROOT}/${refNumber}${ALL_SECTIONS}`;
+        cy.assertUrl(expectedUrl);
+      });
+
+      // go back to the dashboard
+      header.navigation.applications().click();
+
```

Comment:
Can we update this test, the description says it should redirect to the dashboard, but this is not whats happening and the test now manually navigates back to the dashboard

Reply:
updated it description

Reply:
updated it description